clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -699,28 +642,17 @@ runSYCLPostLinkTool(ArrayRef<StringRef> InputFiles, const ArgList &Args) {
     return Modules;
   }
 
-  return parseSplitModulesFromFile(*TempFileOrErr);
-}
-
-Error printSplitModules(const std::vector<module_split::SplitModule> &Modules) {
```

Comment:
Where has this been moved to? I do not see it in the ModuleSplitter. Can you please check? Thanks